app/src/main/res/values-nl/strings.xml

```diff
@@ -13,7 +13,7 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
-    <string name="app_title">Simple Draw Pro - The app for your quick sketches</string>
+    <string name="app_title">Eenvoudig Tekenen - Dé app voor snelle schetsen</string>
```

Comment:
51 chars with Pro. I could replace " - " with ": ", but removing 1 char would help for consistency